### ISSUE TITLE
Unknown type when using "identity"-type-functions on ParamSpec args/kwargs

### ISSUE THREAD

**Issue by diceroll123** [microsoft/pyright#5845]
**Describe the bug**
Using `copy.copy` or similar "identity"-type-functions on ParamSpec args/kwargs reveals the type as `Unknown`

**Code or Screenshots**
I'll just implement my issue into the ParamSpec example straight from the docs.

```python
from collections.abc import Callable
from typing import TypeVar, ParamSpec
from copy import copy

T = TypeVar('T')
P = ParamSpec('P')

def noop(v: T) -> T:
    return v

def add_logging(f: Callable[P, T]) -> Callable[P, T]:
    def inner(*args: P.args, **kwargs: P.kwargs) -> T:
        reveal_type(copy({"a": 1}))  # dict[str, int] (correct, for demonstration purposes)

        reveal_type(args)  # P.args (correct)
        reveal_type(kwargs)  # P.kwargs (correct)

        reveal_type(copy(args))  # Unknown

        reveal_type(noop(kwargs))  # Unknown
        reveal_type(copy(kwargs))  # Unknown
        reveal_type(kwargs.copy())  # dict[str, object] (correct, I guess...?)
        return f(*args, **kwargs)
    return inner

@add_logging
def add_two(x: float, y: float) -> float:
    '''Add two numbers together.'''
    return x + y
```

**VS Code extension or command-line**
Running on Pylance `v2023.8.41`

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.325, which I just published. It will also be included in an upcoming pylance release.
